types/src/model/util.rs

```diff
@@ -4,3 +4,35 @@
 //!
 //! These structs model the types returned by the JSON-RPC API but have concrete types
 //! and are not specific to a specific version of Bitcoin Core.
+
+use serde::{Deserialize, Serialize};
+
+/// Models the result of JSON-RPC method `createmultisig`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct CreateMultisig {
+    /// The value of the new multisig address.
+    pub address: String,
+    /// The string value of the hex-encoded redemption script.
+    #[serde(rename = "redeemScript")]
+    pub redeem_script: String,
+}
+
+/// Models the result of JSON-RPC method `validateaddress`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ValidateAddress {
+    /// If the address is valid or not. If not, this is the only property returned.
+    pub isvalid: bool,
+    /// The bitcoin address validated.
+    pub address: Option<String>,
+    /// The hex encoded scriptPubKey generated by the address.
+    #[serde(rename = "scriptPubKey")]
```

Comment:
We don't need these in `model` because we control the field names. These serde renames are because Core controls the field names when doing the RPC calls but the model types are never used directly against Core.

Reply:
Sometimes we need `serde(with ...` which is used by the serde derives to know how to de/serialize.

In explanation. The sered traits are implemented for `model` types so that users of the lib can store (eg `model::GetTransaction` in a data base or send it over the wire. However these types are never meant to be sent to Core. Since the types in e.g., `types/src/v17/foo.rs` are received over the wire from Core the stuff in there is to support that use case.